app/utils/utils.ts

```diff
@@ -221,6 +232,7 @@ export const adobeAnalyticsRouteToPageName = (route = "", queryParams = "") => {
  * Tracks a virtual page view to Adobe Analytics on page navigation.
  */
 export const trackVirtualPageView = (pathname = "") => {
+  // @ts-ignore
   const adobeDataLayer = window["adobeDataLayer"] || [];
```

Comment:
This one's really gnarly– tried `any`, etc, tried `interface AdobeDataLayerEvent {
  event?: string;
  page_name?: string;
  site_section?: string;
}`... open to ideas

Reply:
honestly, I don't want to get too involved with Adobe and getting the type right for it. So this is fine, but can you add a comment above the `ts-ignore` and say something along the lines that we're just following Adobe's instructions and they don't provide ts support.